src/kv/kvtypes.h

```diff
@@ -61,21 +61,35 @@ namespace kv
   class TxHistory
   {
   public:
-    using RequestID = std::tuple<size_t /* Caller ID */, size_t /* Client Session ID */, kv::Version>;
+    using RequestID = std::tuple<
+      size_t /* Caller ID */,
+      size_t /* Client Session ID */,
+      kv::Version>;
```

Comment:
There is an inconsistency here between the third element of the tuple: here defined as a `kv::Version` but in `frontend.h`, we initialise it to the JSON-RPC sequence number (`reqid = {caller_id.value(), ctx.client_session_id, jsonrpc_id};`)

Reply:
Yes, this is definitely wrong.

Reply:
Yes thank you, I will update this